<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>996</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заполнение областей</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="view">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>621</width>
      <height>671</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>351</width>
      <height>671</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="table">
       <property name="cursor" stdset="0">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <attribute name="horizontalHeaderMinimumSectionSize">
        <number>54</number>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>174</number>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>35</number>
       </attribute>
       <column>
        <property name="text">
         <string>X</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Y</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Радиус:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="radius">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>310</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="add_circle">
       <property name="text">
        <string>Добавить окружность</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="x">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>621</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="y">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>671</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="addpoint">
       <property name="text">
        <string>Добавить точку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="delay">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>С задержкой</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="lock">
       <property name="text">
        <string>Замкнуть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="add_seed_point">
       <property name="text">
        <string>Ввести затравочную точку</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="paint">
       <property name="text">
        <string>Закрасить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PickColor">
       <property name="text">
        <string>Выбрать цвет</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="time">
       <property name="text">
        <string>Измерить время</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="erase">
       <property name="text">
        <string>Очистить</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
